toml/src/main/kotlin/org/rust/toml/Util.kt

```diff
@@ -92,6 +92,19 @@ class StringValueInsertionHandler(private val keyValue: TomlKeyValue) : InsertHa
     }
 }
 
+/** Wraps inserted string with quotes if needed */
+class StringLiteralInsertionHandler : InsertHandler<LookupElement> {
+    override fun handleInsert(context: InsertionContext, item: LookupElement) {
+        val leaf = context.getElementOfType<PsiElement>() ?: return
+        val hasQuotes = leaf.parent is TomlLiteral && leaf.elementType != TomlElementTypes.NUMBER
```

Comment:
Do I understand right that everything inside `TomlLiteral` is either string or number? Or what about other things like booleans? `hasQuotes` looks a bit confusing to me (maybe it works fine, but just looks a bit strange)

Reply:
Hmm, this was just copied from somewhere... Yes, I think it's better to write `val hasQuotes = leaf.parent is TomlLiteral && leaf.elementType in TOML_STRING_LITERALS` in this case